### ISSUE TITLE
html-req-lang fails when lang is present

### ISSUE THREAD

**Issue by waffledonkey** [htmllint/htmllint#102]
It seems as though `element.hasOwnProperty("lang")` returns false even when the attribute is present.  `element.attribs.lang` returns true and may be a better option.

```html
$head -2 index.html
<!DOCTYPE html>
<html lang="en" class="no-js">
```
```
$ htmllint index.html
index.html: line 2, col 1, html element should specify the language of the page
```

**Comment by mlochbaum** [comment]
It takes some sneaky coding to not write any tests for a file and still get 100% test coverage... I'll change it to match img-req-alt and img-req-src.
